Bash_scripts/HW20/group_vars/ubuntu/ubuntu_var.yml

```diff
@@ -0,0 +1,5 @@
+#Ubuntu vars
+group_hw20: ZONE_1
+group_sudo: sudo,docker
+src_curl: /home/engelko/DevOps_course/Bash_scripts/HW20/roles/hw202/files/curl_7.68.deb
+dest_curl: /tmp/curl_7.68.deb
```

Comment:
var name have no sense

Reply:
the full path meant? I don't quite understand what you mean